### PR TITLE
Update for the confusion-table

### PR DESCRIPTION
**Pull request by TillMeineke**
I still think the table looks more accurate in my version

### CODE REVIEW

File: 04-evaluation/03-confusion-table.md [hunk 1]
```diff
@@ -14,57 +11,49 @@ Confusion table is a way of measuring different types of errors and correct deci
...
 
-The **accuracy** corresponds to the sum of TN and TP divided by the total of observations. 
+The **accuracy** corresponds to the sum of TN and TP divided by the total of observations.
 
-The code of this project is available in [this jupyter notebook](notebook.ipynb).  
+The code of this project is available in [this jupyter notebook](https://github.com/alexeygrigorev/mlbookcamp-code/blob/master/course-zoomcamp/04-evaluation/notebook.ipynb).
 
 Add notes from the video (PRs are welcome)
 
-<table>
```
Comment (alexeygrigorev) [inline 04-evaluation/03-confusion-table.md]: can you please undo this change?

File: 04-evaluation/03-confusion-table.md [hunk 2]
```diff
@@ -1,11 +1,8 @@
+# 4.3 Confusion table
 
-## 4.3 Confusion table
-
-<a href="https://www.youtube.com/watch?v=Jt2dDLSlBng&list=PL3MmuxUbc_hIhxl5Ji8t4O6lPAOpHaCLR"><img src="images/thumbnail-4-03.jpg"></a>
```
Comment (alexeygrigorev) [inline 04-evaluation/03-confusion-table.md:4]: Can you please undo this change?

File: 04-evaluation/03-confusion-table.md (hunk 1 above)
Comment (TillMeineke) [reply 04-evaluation/03-confusion-table.md]: sure. done

File: 04-evaluation/03-confusion-table.md (hunk 2 above)
Comment (TillMeineke) [reply 04-evaluation/03-confusion-table.md:4]: sure. done
